app/controllers/skills_controller.rb

```diff
@@ -0,0 +1,48 @@
+class SkillsController < ApplicationController
+  before_action :authenticate_user!, only: [:destroy, :delete, :new, :create, :edit, :update]
+  autocomplete :skill, :name
+
+  before_action :get_skill, only: [:edit, :update, :destoy]
+
+  def get_skill
+    @skill = Skill.find(params[:id])
+  end
+
+  def new
+    @skill = Skill.new
+  end
+
+  def create
+    @skill = Skill.new(skill_params)
+    if current_user
+      if @skill.save
+       redirect_to skill_path(@skill), notice: "Skill sucessfully created!"
+      else
+        render "new"
+      end
+    end
+  end
+
+  def edit
+
+  end
+
+  def update
+    @skill.update(skill_params)
+  end
+
+  def destroy
+  #only accesible by admin
+  #if current user is admin statement
```

Comment:
Incorrect indentation detected (column 2 instead of 4).<br>Missing space after #.